README.md

```diff
@@ -18,19 +18,232 @@ Alternatively, you can use the Open in Colab browser extension ([Chrome](https:/
+| path              | text    | The JSON Pointer of the field to which this count relates, excluding array indices.                            |
+| path_array        | text[]  | The components of the JSON Pointer of the field to which this count relates.                                   |
+| object_property   | integer | The number of occurrences of the field, across all array entries and all releases.                             |                                                                          |
+| array_count       | integer | If the field is an array, the cumulative length of its occurrences, across all array entries and all releases. |                                                                          |
+| distinct_projects | integer | The number of distinct projects in which the path appears.                                                     |
+
+#### project_fields
+
+| Column        | Type    | Description                                             |
+| ------------- | ------- | ------------------------------------------------------- |
+| collection_id | integer | The collection to which the project belongs.            |
+| project_id    | text    | The `id` of the project.                                |
+| paths         | text[]  | The JSON Pointers of the fields present in the project. |
+
+#### check_results
+
+| Column        | Type              | Description                                                                                                                  |
+| ------------- | ----------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| run_id        | character varying | The run in which this result was calculated.                                                                                 |
+| check_id      | character varying | The check that this result relates to.                                                                                       |
+| result        | boolean           | The result of the check. TRUE represents success, FALSE represents failure and NULL indicates that no result was calculated. |
+| output        | jsonb             | The output of the check.                                                                                                     |
+| collection_id | integer           | The collection to which this check result relates.                                                                           |
+
+#### run_collection
+
+| Column        | Type              | Description                               |
+| ------------- | ----------------- | ----------------------------------------- |
+| run_id        | character varying | A run of data quality checks.             |
+| collection_id | integer           | A collection that was checked in the run. |
```

Comment:
```suggestion
| collection_id | integer           | The collection that was checked in the run. |
```
sorry missed this one the last time, all the other similar descriptions refer to "The collection"

Reply:
I phrased it like that because, although each row in this table relates a collection to a run, many collections can be checked in one run.